SpatialGDK/Source/SpatialGDKFunctionalTests/Private/TestMaps/SpatialRPCTimeoutMap.cpp

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#include "TestMaps/SpatialRPCTimeoutMap.h"
+#include "GameFramework/PlayerStart.h"
+#include "SpatialGDKFunctionalTests/SpatialGDK/SpatialTestRPCTimeout/SpatialTestRPCTimeout.h"
+#include "SpatialGDKFunctionalTests/SpatialGDK/SpatialTestRPCTimeout/SpatialTestRPCTimeoutGameMode.h"
+
+USpatialRPCTimeoutMap::USpatialRPCTimeoutMap()
+	: UGeneratedTestMap(EMapCategory::CI_PREMERGE, TEXT("SpatialRPCTimeoutMap"))
+{
+	// clang-format off
+	SetCustomConfig(TEXT("[/Script/SpatialGDK.SpatialGDKSettings]") LINE_TERMINATOR
+		TEXT("QueuedIncomingRPCWaitTime=0"));
+	// clang-format on
+
+	SetNumberOfClients(2);
+
+	EnableMultiProcess();
+}
+
+void USpatialRPCTimeoutMap::CreateCustomContentForMap()
+{
+	ULevel* CurrentLevel = World->GetCurrentLevel();
+	ASpatialWorldSettings* WorldSettings = CastChecked<ASpatialWorldSettings>(World->GetWorldSettings());
+	WorldSettings->DefaultGameMode = ASpatialTestRPCTimeoutGameMode::StaticClass();
+
+	FTransform Transform1 = FTransform::Identity;
+	Transform1.SetLocation(FVector(-300.f, -500.f, 200.f));
```

Comment:
Can you comment what this specific location is used for?

Reply:
The comment for this is just below above the player start (which uses this location). Should I move it up?

Reply:
Ah that's aight